File: engine/model/src/main/scala/tools/packet/guild/GuildNameChange.scala
package tools.packet.guild

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class GuildNameChange(private var _characterId: Int, private var _guildName: String) extends PacketInput {
  def characterId: Int = _characterId

  def guildName: String = _guildName

  override def opcode(): SendOpcode = SendOpcode.GUILD_NAME_CHANGED
}

File: engine/engine-model/src/main/java/tools/packet/guild/GuildNameChange.java
package tools.packet.guild;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record GuildNameChange(Integer characterId, String guildName) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.GUILD_NAME_CHANGED;
   }
}